liferay-frontend/liferay-portal
# LPS-176053 Research the needs for the client extension for cell renderer

> :warning: Before testing this PR read the [confluence doc](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/2283896840/LPS-176053+-+Research+the+needs+for+the+client+extension+for+cell+renderer).

Steps to test this PR:

1. Download the PR.
2. Build (ant all) the PR.
3. Deploy project `frontend-data-set-sample-web` of `liferay-portal` with `gradle deploy -a`. This is necessary since it is not deployed by default.
4. Start portal.
5. Download the attached [.tar.gz](https://github.com/liferay-frontend/liferay-portal/files/10848102/devexp-ng-workspace.tar.gz) blade workspace.
6. Unpack the `.tar.gz` file.
7. Run `yarn && yarn prepare` in the expanded workspace. That will install all dependencies and copy the `frontend-js-api` dependency to `node_modules`, since it is not yet published to the npm registry.
8. Run `yarn build` inside the workspace's `packages/ce-sample` directory.
9. Deploy the built `ce-sample.zip` file inside the `dist` directory of the project to the running portal.
10. Edit the home page and add the FDS sample portlet to the page.
11. Go to the customized tab of the FDS sample portlet and see the 🍏s in the table. Those are being rendered by the CE.

![image](https://user-images.githubusercontent.com/3273630/221826882-8b709840-bdd7-4c1a-9f44-cb48e1405cad.png)
